### ISSUE TITLE
Questions regarding user specific data

### ISSUE THREAD

**Issue by Kristofferson777** [alcantarar/Recurrent_GRF_Prediction#7]
I have a question regarding setting up the data needed for GRF prediction based on user specific data. Regarding the acceleration data such as Sample_test_data.csv, I would like to know how that should be formatted. Is there a specific part of the leg that should be identified as the shank and what are the formatting requirements for the .csv. Regarding formatting requirements is there a sampling rate range, can multiple shanks be added...etc? Regarding footstrike I was hopping you could let me know how to create a footstrike file that the program can use based of acceleration data, a video, and subject information. Lastly, how can I get the grf data to be outputted as a excel file, or .trc, or csv?

In summary I basically would like to know how to format my files and data for the program if I have raw acceleration data, a video of the movement for two angles, and subject info like height and weight. I'd also like to know how to get the grf prediction results in a raw data format like excel, csv, or .trc. 

Thank you for your time.

**Comment by alcantarar** [comment]
>Regarding the acceleration data such as Sample_test_data.csv, I would like to know how that should be formatted.

I'm unclear on what your question is. `Sample_test_data.csv` is the example for how to format the data you'd like to test the prediction model with. `Test_LSTM.ipynb` utilizes this data file and provides more info about what it contains.

>Is there a specific part of the leg that should be identified as the shank and what are the formatting requirements for the .csv.

`README.md` links to the manuscript with information about the location of the accelerometers.

>Regarding formatting requirements is there a sampling rate range, can multiple shanks be added...etc? 

Sure, but you'll need to retrain your prediction model to utilize this added signal. 

>Regarding footstrike I was hopping you could let me know how to create a footstrike file that the program can use based of acceleration data, a video, and subject information.

I think this is beyond the scope of a github issue. 

> Lastly, how can I get the grf data to be outputted as a excel file, or .trc, or csv?

The model output can already be exported to any text file format you'd like.
